### PR TITLE
LPS-119069 baseline

### PR DESCRIPTION
**Pull request by marcosapmf**
Related Issue: https://issues.liferay.com/browse/LPS-119069

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:app-builder

**Comment by natocesarrego** [comment]
@marcosapmf, are the failures reported by App Builder suite unrelated to your pull?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Dynamic Data Mapping API
 Bundle-SymbolicName: com.liferay.dynamic.data.mapping.api
-Bundle-Version: 8.3.2
+Bundle-Version: 8.4.0
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd:3]: @marcosapmf, let's send to Brian, but be aware that maybe he can reject it due to the release's proximity.

Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd:3]: @natocesarrego, in this case, shouldn't we hold this until the release is validated?

**Comment by marcosapmf** [comment]
> @marcosapmf, are the failures reported by App Builder suite unrelated to your pull?

@natocesarrego, from what I've seen, I don't think those errors are related to this pull, since my changes regard to the Rules and errors regard to an error message and the Users and Organizations menu navigation. Morover, I'll fix the rebase conflicts and the commit message you pointed above and then run the app-builder suite again so that we can analyse it better. 
Thanks!

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:app-builder

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd (hunk 1 above)
Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd]: @marcosapmf, we can send it and see how it goes or we can talk to him on Slack, which one do your prefer?

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:app-builder

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd (hunk 1 above)
Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd]: @natocesarrego, so I think we can send it :)

**Comment by natocesarrego** [comment]
ci:test:app-builder

**Comment by natocesarrego** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
@marcosapmf, after last CI results, I think we're good to go. If you agree, please run `ci:forward`.

Thanks.

**Comment by marcosapmf** [comment]
ci:forward

**Pull request closed without merging**